bin/registerServerTopology.sh

```diff
@@ -490,7 +490,8 @@ parse_command_options "$@"
 if [[ ! $TEMPLATE_FILE ]];
 then
   # Ask for API TARGET Password
-  get_api_password "$TARGET_NODE_ADDRESS"
+  echo "$TARGET_NODE_ADDRESS : Please enter your password:"
```

Comment:
```suggestion
  echo "Please enter the password of $TARGET_NODE_ADDRESS:"
```

Reply:
I would use the -n option of `echo` command so that everything stills on the same line

`Please enter the password of the 192.168.1.20: centreon`

```suggestion
    echo -n "Please enter the password of $TARGET_NODE_ADDRESS: "
```